instrumentation/servlet/servlet-common/bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/servlet/InjectionState.java

```diff
@@ -8,7 +8,7 @@
 // this is shared by both ServletOutputStream and PrintWriter injection
 public class InjectionState {
   private static final int HEAD_TAG_WRITTEN_FAKE_VALUE = -1;
-  private static final int HEAD_TAG_LENGTH = "<head>".length();
+  private static final int HEAD_TAG_PREFIX_LENGTH = 4;
```

Comment:
You could use the length of the string here
```suggestion
  private static final int HEAD_TAG_PREFIX_LENGTH = "<head".length();
```
for a better readability, and...